src/Compilers/CSharp/Portable/Binder/Binder.ValueChecks.cs

```diff
@@ -1138,6 +1398,16 @@ static private ErrorCode GetStandardLvalueError(BindValueKind kind)
             throw ExceptionUtilities.UnexpectedValue(kind);
         }
 
+        static private ErrorCode GetStandardRValueRefEscapeError(uint escapeTo)
+        {
+            if (escapeTo == Binder.ExternalScope)
+            {
+                return ErrorCode.ERR_RefReturnLvalueExpected;
```

Comment:
Could you use a different error code here? This one is already used above.
This will also help review, as it will be obvious which unittest hits this path. Thanks

Reply:
I could actually use ERR_EscapeOther here unconditionally. It would be valid error, but I think the message for ERR_RefReturnLvalueExpected fits better. If I maka another error for this it would still report the same error as in ERR_RefReturnLvalueExpected. I am not sure it is worth it.

---
In reply to: [135400818](https://github.com/dotnet/roslyn/pull/21334#discussion_r135400818) [](ancestors = 135400818)